cmd/create/cluster/cmd.go

```diff
@@ -862,6 +881,41 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	enableCustomerManagedKey := args.enableCustomerManagedKey
+
+	if interactive.Enabled() {
```

Comment:
I think this should be
```suggestion
	if interactive.Enabled() && kmsKeyARN == "" {
```
Since if the user explicitly set `kmsKeyARN` then we don't need to confirm that they want it.